huaweicloud/resource_huaweicloud_cce_cluster_v3.go

```diff
@@ -382,6 +383,15 @@ func resourceCCEClusterV3Create(d *schema.ResourceData, meta interface{}) error
 	}
 	d.SetId(create.Metadata.Id)
 
+	icAgentClient, err := config.AomV1Client(GetRegion(d, config))
+
+	installParam := icagents.InstallParam{
+		ClusterId: d.Id(),
+		NameSpace: "default",
+	}
+
+	_ = icagents.Create(icAgentClient, installParam)
```

Comment:
suggest to check the result and log the error message.

Reply:
fixed